src/receta/controller/Main.java

```diff
@@ -4,75 +4,75 @@
  * and open the template in the editor.
  */
 package receta.controller;
+import receta.*;
 
-import receta.Ingrediente;
-import receta.Receta;
 
 /**
  *
  * @author jpramire
  */
 public class Main {
-    /*private String nombre;
-    private String intrucciones;   
-    private String autor; 
-    private Ingrediente [] ingrediente = new Ingrediente[5];*/
    
   
 
-    /*public static void main(String[] args) {
-       Receta empanadaObj = new Receta(); // nombre de la instancia empanadaObj
-       Receta ajiaco = new Receta();
+    public static void main(String[] args) {/*
+    
+    /******************************************HOSPITAL***********************************************/
+    Cocina hospital = new Cocina(5); 
+    hospital.setNombreCocina("Childrens' Hospital");
+    hospital.setUbicacionCocina("Cincinnati,OH");
+    hospital.setAliasCocina("Primer piso");    
+    
+    Carne ihospital1 = new Carne();
+    ihospital1.setNombre("Pollo");
```

Comment:
Pregunta... tengo que poner getters and setters para todas las variables privadas? por ejemplo en el caso de los paramentros debo ponerlos tambien?